artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -643,4 +645,18 @@
     "Cisco Secure Desktop installation panel was detected.": "wykryto panel instalacyjny Cisco Secure Desktop.",
     "Openfire Admin Console login panel was detected.": "wykryto panel logowania konsoli administracyjnej Openfire.",
     "MinIO Browser login panel was detected.": "wykryto panel logowania MinIO Browser.",
+    "noVNC login panel was detected.": "wykryto panel logowania noVNC.",
```

Comment:
Wygląda na pełne zdania, więc pewnie wszystko powinno być z wielkiej? No ale to niekoniecznie problem z tym PR tylko ogólnie.